src/windows/export.py

```diff
@@ -179,6 +179,34 @@ def __init__(self):
             self.channel_layout_choices.append(layout[0])
             self.cboChannelLayout.addItem(layout[1], layout[0])
 
+        # Add streamable file format presets (mp4/mov only)
+        #
+        # 0 - none
+        # 1 - faststart
+        # 2 - fragmented
+        #
+        streamable_presets = [_("Disable"), _("Place index table to the beginning"), _("Make selfcontained fragmented file")]
+        for streamable in streamable_presets:
+            # append preset to list
+            self.cboStreamablePreset.addItem(streamable)
+
+        self.cboStreamablePreset.setToolTip(_("<b>Disable:</b><br>"
+                                              "Export media to regular not streamable file format<br>"
+                                              "<br>"
+                                              "<b>Place index table to the beginning:</b><br>"
+                                              "Runs a second pass moving the index ('moov' box) to the beginning of the file. "
+                                              "Requires more disk operations to complete.<br>"
+                                              "FFmpeg's equivalent of <i>-movflags faststart</i><br>"
+                                              "<br>"
+                                              "<b>Make selfcontained fragmented file:</b><br>"
+                                              "Writes the index table as fragments across the file. Creates fragments that at least 8 seconds long. "
+                                              "Requires modern software for playback and edit.<br>"
+                                              "FFmpeg's equivalent of <i>-movflags frag_keyframe -min_frag_duration 8000000</i>"))
```

Comment:
Aside from the localization...

1. I wouldn't put the FFmpeg arguments in italics, it's hard to read. Qt's rich text markup supports `<tt>` for monospace text, I think that's a better choice. Or perhaps even `<code>`, if that works.
2. I found that the markup also supports HTML lists, including `<dl>` which is practically tailor-made for this sort of thing.

I found that this markup:
```python
….setToolTip(_("<dl><dt><b>Disable</b></dt>"
               "<dd>Export media to regular not streamable file format</dd>"
               "<dt><b>Place index table to the beginning</b></dt>"
               "<dd>Runs a second pass moving the index ('moov' box) to the beginning of the file. Requires more disk operations to complete.<br>"
               "FFmpeg's equivalent of <tt>-movflags faststart</tt></dd>"
               "<dt><b>Make selfcontained fragmented file</b></dt>"
               "<dd>Writes the index table as fragments across the file. Creates fragments that at least 8 seconds long. Requires modern software for playback and edit.<br>"
               "FFmpeg's equivalent of <tt>-movflags frag_keyframe -min_frag_duration 8000000</tt></dd></dl>"))
```
...renders properly, and is more compact without sacrificing readability (IMHO):
![Openshot_export_tooltip](https://user-images.githubusercontent.com/538020/61416547-0acf7280-a8c2-11e9-9914-4452d45fa61f.png)

Like I said, for the most part each line should be a separate translation string, _without_ the markup, and with the `<tt>...</tt>` blocks not translated at all. I'd also set something like `<tt style='white-space:pre'>` for the argument blocks, so that they don't get wrapped.

#### Minor nits
1. "FFmpeg's equivalent of" isn't correct, _our_ option is actually the _equivalent of_ the FFmpeg arguments listed. (Or, the arguments are the "FFmpeg equivalent" for our option.) That second one would work ("FFmpeg equivalent:"), or "FFmpeg flags:", "FFmpeg options:", something like that.
2. "Place index table to the beginning" should be "Place index table **at** the beginning". It's "moved to" the beginning, at which point it's located "at" the beginning.
3. I would actually be more descriptive of the _result_, less of the _action_, for all options. IOW:
    * `disable`: The file is not optimized for streaming.
    * `faststart`: The entire index is located at the start of the file. Playback can start before the file is finished downloading. Encoding requires additional processing, and will be slower.
    * `fragmented`: The index is broken into pieces and placed at intervals (at least 8 seconds apart) throughout the file. Playback can start quickly from any point in the file. Can only be played by software which has support for this feature; many do not.

Reply:
>... Encoding requires additional processing, and will be slower...

Nope, it is fast as your hard drive's internal copy of the large data block (just few seconds for large files). But this routine rewrites cells where info stored, so you aware of the impact it may cause on SSD. If final file size 4GB - about 4GB will be rewritten.

About grammar - yeah, I need to fix it. Thank you.

Reply:
> Nope, it is fast as your hard drive's internal copy of the large data block (just few seconds for large files). 

Interesting. Well, that's good to hear.

If there's really no impact to the user (I don't consider one set of extra writes to be impact, personally) then I don't think we need to trouble them with that kind of detail. Half of them are probably running with a paging file on their SSD anyway.